{{cookiecutter.repo_name}}/.github/workflows/gh-ci.yaml

```diff
@@ -47,6 +43,7 @@ jobs:
         matrix:
           os: [macOS-latest, ubuntu-latest, windows-latest]
           python-version: ["3.8", "3.9", "3.10"]
+          mdanalysis-version: ["2.1.0", "latest", "develop"]
```

Comment:
We don't really guarantee full compatibility across two versions, it would be fine to just do latest and develop here and save some trees imho